README.md

```diff
@@ -525,6 +525,29 @@ axios.get('/user/12345', {
 source.cancel('Operation canceled by the user.');
 ```
 
+```js
+var CancelToken = axios.CancelToken;
```

Comment:
Can we merge this and previous snippets into one to keep the documentation short? I would insert `axios.post()` (without error handling) between `axios.get()`  and `source.cancel()`.